policy/release/provenance_materials.rego

```diff
@@ -0,0 +1,79 @@
+#
+# METADATA
+# title: Provenance Materials
+# description: >-
+#   This package provides rules for verifying the contents of the materials section
+#   of the SLSA Provenance attestation.
+#
+package policy.release.provenance_materials
+
+import future.keywords.contains
+import future.keywords.if
+import future.keywords.in
+
+import data.lib
+import data.lib.tkn
+
+# METADATA
+# title: Task git-clone missing
+# description: >-
+#   The attestation must contain a git-clone task with the expected commit and url results.
+# custom:
+#   short_name: missing_git_clong_task
+#   failure_msg: Task git-clone not found
+#   collections:
+#   - minimal
+#
+deny contains result if {
+	some attestation in lib.pipelinerun_attestations
+	not tkn.git_clone_task(attestation)
+	result := lib.result_helper(rego.metadata.chain(), [])
+}
+
+# METADATA
+# title: Git repo materials mismatch
+# description: >-
+#   The result of the git-clone task must be included in the materials section of the SLSA
+#   provenance attestation.
+# custom:
+#   short_name: git_repo_materials_mismatch
+#   failure_msg: Entry in materials for the git repo %q at commit %q not found
```

Comment:
Replace `at` with `and`?

Reply:
or maybe `or` since it's checking for both

Reply:
Thanks! I changed it to `and` because both must be present.